apps/molgenis-components/src/components/forms/InputRefItem.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <input
+    v-if="rowKey"
+    :id="`${id}-${rowAsString}`"
+    :name="`${id}-${rowAsString}`"
+    type="radio"
+    :checked="isSelected"
+    @change="$emit('update:modelValue', rowKey)"
+    class="form-check-input"
+    :class="{ 'is-invalid': errorMessage }"
+  />
+  <label class="form-check-label" :for="`${id}-${rowAsString}`">
+    {{ rowAsString }}
+  </label>
+</template>
+
+<script>
+import BaseInput from "./baseInputs/BaseInput.vue";
+import { flattenObject } from "../utils";
+
+export default {
+  name: "InputRefItem",
+  extends: BaseInput,
+  props: {
+    id: { required: true, type: String },
+    tableName: { required: true, type: String },
+    row: { required: true, type: Object },
+    selection: { required: false, type: Object },
+    client: { required: true, type: Object },
+  },
+  data() {
+    return {
+      rowKey: "",
+      isSelected: this.rowKey?.name === this.selection?.name,
+    };
+  },
+  computed: {
+    rowAsString() {
+      return flattenObject(this.rowKey).trim();
+    },
+    isSelected() {
+      return this.rowKey?.name === this.selection?.name;
+    },
```

Comment:
Can't have this as both computed and in data. Just computed should be enough here I think.

Reply:
removed it from data